cli/src/config/revsets.toml

```diff
@@ -9,6 +9,9 @@ simplify-parents = "reachable(@, mutable())"
 # suppress symbol resolution error.
 log = "present(@) | ancestors(immutable_heads().., 2) | present(trunk())"
 sign = "reachable(@, mutable())"
+# Emit the working-copy branch first, which is usually most interesting.
+# This also helps stabilize output order.
+log-graph-prioritize = "present(@)"
```

Comment:
nit: move this up one line? it's closely related to `revsets.log`

Reply:
Done